Polyhedron/demo/Polyhedron/Scene.cpp

```diff
@@ -245,6 +268,8 @@ Scene::drawWithNames(Viewer_interface* viewer)
 void 
 Scene::draw_aux(bool with_names, Viewer_interface* viewer)
 {
+    if(!ms_splatting->viewer_is_set);
+        ms_splatting->setViewer(viewer);
```

Comment:
There is a bug, there. Hopefully, [there was a compiler smart enough to warn about it][1]:
```
/home/cgal_tester/build/src/cmake/platforms/ArchLinux-clang-Release/test/Polyhedron_Demo/Scene.cpp:271:37: warning: if statement has empty body [-Wempty-body]
    if(!ms_splatting->viewer_is_set);
                                    ^
```
[1]: https://cgal.geometryfactory.com/CGAL/Members/testsuite/CGAL-4.7-Ic-134/Polyhedron_Demo/TestReport_lrineau_ArchLinux-clang-Release.gz